### DISCUSSION TITLE
求sepoliaeth！！！

### DISCUSSION THREAD

**Discussion by barry-hou** [smartcontractkit/Web3_tutorial_Chinese#26] [Q&A] [answered]
兄弟萌，求sepoliaeth！！！ 地址:0xF402A7B0bA75CEa0188D0cC00B6f62Fff6b4B464

**Comment by romandorian** [comment]
同样求seploliaeth：0x8345fec9c3491De90b48079C65Dbc60D6552dD34

**Comment by liuxulife** [comment]
同 sepoliaeth : 0xDc900033937A6957174070e9721a4113Dd0E0D84

**Comment by dadadong** [comment]
同求 0x5C5B9a73cA55602064EB295Cde5dE1eC5bffD994

**Comment by ghost** [comment]
同求 0x5b5eE8A57017624A015a269c4022dEe8B67516b5

**Comment by QingyangKong** [comment, marked as answer]
在 B站视频下方评论，会定期打
